components/hab/src/command/studio/docker.rs

```diff
@@ -37,6 +37,10 @@ const DOCKER_SOCKET: &'static str = "/var/run/docker.sock";
 const HAB_STUDIO_SECRET: &'static str = "HAB_STUDIO_SECRET_";
 
 pub fn start_docker_studio(_ui: &mut UI, mut args: Vec<OsString>) -> Result<()> {
+    if args[0] == OsString::from("rm") {
```

Comment:
You generally want to avoid the index `[…]` operator in rust, as [it can panic](http://doc.rust-lang.org/1.28.0/alloc/vec/struct.Vec.html#indexing).

A good alternative is the [`get` operation provided by the slice type](http://doc.rust-lang.org/1.28.0/std/primitive.slice.html#method.get) (which  a [Vec can deref to implicitly](http://doc.rust-lang.org/1.28.0/alloc/vec/struct.Vec.html#method.deref)), which returns an [`Option`](http://doc.rust-lang.org/1.28.0/core/option/enum.Option.html).